pandas/core/frame.py

```diff
@@ -7440,7 +7440,20 @@ def transform(self, func, axis=0, *args, **kwargs) -> "DataFrame":
         axis = self._get_axis_number(axis)
         if axis == 1:
             return self.T.transform(func, *args, **kwargs).T
-        return super().transform(func, *args, **kwargs)
+
+        if isinstance(func, list):
```

Comment:
instead call a function in pandas/core/aggregation.py (alt *could* create pandas/core/aggregation/transforms.py) - e.g. make a sub-module is ok; but if doing that pls do as a pre-cursor to move things.